### ISSUE TITLE
Liquid cooling

### ISSUE THREAD

**Issue by jordens** [sinara-hw/Booster#395]
Let's evaluate designing a liquid cooling interface as an alternative to the fins and fans.
@gkasprow as discussed 

We had a mechanics repo but I think that's dormant/unused.

**Comment by gkasprow** [comment]
I see 2 options. Modify the original design and instead of fins drill holes and install plugs. I did such things in the past![2014-11-17 08.07.26.jpg](https://github.com/sinara-hw/Booster/assets/4325054/5a964cc2-b9c9-41f2-a56f-7c681596213d)

It's optimal from thermal point of view

 Second option is to attach COTS liquid cold plates used in PCs to cool chipsets, RAM, SSDs etc. They are cheap but the design changes often with chipset evolution

I'd like to do it in elegant way using quick disconnect no drip connectors from Koolance. They are affordable and good.
One could replace the modules witout emptying the circuit.

**Comment by gkasprow** [comment]
Another issue is how we circulate the liquid.
I use COTS radiatots with pump, resevoir, sensors.
It was sth like 200EUR when i bought it in 2015.
Then how we wire the circuits?
Do we connect them in series or in parallel?

Yet another option is big cold plate installed at the bottom of the enclosure and we place all modules on top of it. It could be problematic to make it and maintain optimal thermal resistances.

**Comment by jordens** [comment]
Let's keep the internal and external chiller stuff separate. The external stuff doesn't appear to be so hard to supply and will also be application specific.
A common plate for the modules might also be problematic due to multiple existing mechanical constraints (connectors, existing screws). The gradient and crosstalk of series operation (or even parallel with a common plate) might be acceptable. We should first get a number on the power and phase tempos and time constat though.

**Comment by gkasprow** [comment]
This is the proposed design
![unnamed](https://github.com/sinara-hw/Booster/assets/4325054/426d4dde-0eb5-44ba-a658-34a855b9b875)
![unnamed](https://github.com/sinara-hw/Booster/assets/4325054/8994d087-93d5-4622-85f1-58c6b637081a)

we will swap IEC power and cooling connectors so the latter are below the IEC. External connectors are non-spilt, so no need of draining liquid when changing connectivity.
![unnamed](https://github.com/sinara-hw/Booster/assets/4325054/ff5dab2a-59d8-41fc-8a33-1efe6db62943)

The connectivity (series/parallel/mixed) may change.

**Comment by gkasprow** [comment]
update: Booster cold plate set
![PXL_20240718_165512659 MP](https://github.com/user-attachments/assets/f5e4ff00-9f70-487c-8ccc-f09b1d40b06c)
![PXL_20240718_164925434](https://github.com/user-attachments/assets/5e63783d-6903-4d10-a2a1-4b11099e0ba7)
